Library/Homebrew/sorbet/rbi/homebrew.rbi

```diff
@@ -0,0 +1,30 @@
+# typed: strict
+
+module Homebrew
+  sig { params(blk: T.untyped(T.proc.bind(Homebrew::CLI::Parser).void)).returns(Homebrew::CLI::Parser) }
+  def home_args(&blk)
+  end
+
+  sig { void }
+  def home
+  end
+
+  module CLI
```

Comment:
I wonder if we'd want a `Library/Homebrew/sorbet/rbi/cli/parser.rbi` file for this?